AssemblerLib/MeshComponentMap.cpp

```diff
@@ -75,12 +100,23 @@ MeshComponentMap::MeshComponentMap(
                 // mesh items are ordered first by node, cell, ....
```

Comment:
This is the beginning of an else clause. It probably differs from the if-clause above only by the index calculation, i.e., a single line. Can't the commen parts of the two cases be merged somehow?